Braintree.xcodeproj/project.pbxproj

```diff
@@ -4704,6 +4677,8 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SKIP_INSTALL = YES;
 				SUPPORTS_MACCATALYST = NO;
+				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
```

Comment:
This change probably wasn't intentional?

Reply:
This was generated automagically by Xcode when we added the Swift files. It looks like it exists for `BraintreeSEPADirectDebit`, `BraintreeAmericanExpress`, `BraintreePayPal`, `BraintreeCore`, `BraintreePayPalNativeCheckout`, and `BraintreeDataCollector` (so all of the Swift modules). Looking at the [Apple Swift Docs](https://github.com/apple/swift/blob/main/docs/OptimizationTips.rst#enabling-optimizations) for this it states:
> This is meant for normal development. It performs minimal optimizations and preserves all debug info.

If we want a different optimization level for `Debug` we should likely update the entire SDK.

It's worth noting our `Release` build is set to `-O`: 
> This is meant for most production code. The compiler performs aggressive optimizations that can drastically change the type and amount of emitted code. Debug information will be emitted but will be lossy.

Reply:
Interesting, thanks for the docs link! I wonder what kind of "debug info" specifically you miss out on in -0 mode.